src/app/mat-main-nav/desktop-menu/desktop-menu.component.scss

```diff
@@ -0,0 +1,23 @@
+@import "@angular/material/core/style/_variables";
```

Comment:
definitely want to prepend node_modules references with a tilda `~`

Reply:
done, no wonder removing the undrscore fails, was wondering what the issue was duh